packages/lib-webpack/rollup.config.js

```diff
@@ -0,0 +1,11 @@
+// eslint-disable-next-line import/no-relative-packages
+import { tsLibConfig, dtsLibConfig } from '../common/rollup-configs';
+import pkg from './package.json';
+
+// eslint-disable-next-line no-console
+console.log(`Rollup ${pkg.name}`);
```

Comment:
Yes, I'd keep it :smiley: Rollup itself doesn't print the context directory.

I think this is important in case of a monorepo, where you execute Rollup multiple times on different packages.